### PR TITLE
#19: Showing the data from at least one API call

### PR DESCRIPTION
**Pull request by Abdullahtr249**
…ew component "ActivityList" to App.vue

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,20 +1,24 @@
 <template>
-  <div>
+  <div id=app>
```
Comment (arifjehoh) [inline src/App.vue:2]: Why have you added `id=app`, it wasn't need before...

File: src/components/ActivityList.vue [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    },
+    mounted(){
+        Vue.axios.get('https://www.boredapi.com/api/activity')
+        .then((r) => {
+            this.list=r.activity;
+            console.warn(r.activity)
+        })
+    }
+}
+</script>
```
Comment (arifjehoh) [inline src/components/ActivityList.vue:38]: Make sure you use `2 space` per indentation.

File: src/components/ActivityList.vue [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    name: "ActivityList",
+    data(){
+        return{list:undefined}
+    },
+    mounted(){
+        Vue.axios.get('https://www.boredapi.com/api/activity')
+        .then((r) => {
+            this.list=r.activity;
+            console.warn(r.activity)
+        })
```
Comment (arifjehoh) [inline src/components/ActivityList.vue:35]: Can we export this way of using the API to a Javascript file? For more dynamically usage of the API Calls, just like how we did in the DinnerModel(DishSource)

**Review by arifjehoh — CHANGES_REQUESTED** [review]

File: src/App.vue (hunk 1 above)
Comment (Abdullahtr249) [reply src/App.vue]: fixt. I tried to test the cod

File: src/components/ActivityList.vue (hunk 3 above)
Comment (mhaao) [reply src/components/ActivityList.vue:35]: can you test this first? Because the API key was different and I tried and I hade ERORR   :(

**Review by arifjehoh — APPROVED** [review]
Good work! I Fixed the issue in some of the files.

**Pull request merged**